packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -338,6 +358,12 @@ export default defineComponent({
       const displayDate = formatDateFromHTTP(this.file.mdate, this.$language.current)
       return upperFirst(displayDate)
     },
+    showTags() {
+      return this.capabilities?.files.tags && this.file.tags?.length
```

Comment:
Same here regarding capabilities, please don't use them directly from the store anymore.